tests/tests_default_initscripts.yml

```diff
@@ -1,5 +1,11 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
@a-mere-peasant Looks like your build is failing because you renamed this file and the molecule tests in `molecule/default/molecule.yml` are hardcoded to look for it https://github.com/linux-system-roles/network/blob/51fa1146b4eae53606322dbdca32e1386cce5242/molecule/default/molecule.yml#L26

Reply:
Also, any playbook starting with tests_* is checked by the `ensure_non_running_provider.py` to make sure it complies with the old format of tests (e.g. tests_ethernet.yaml and tests_ethernet_other_provider.yaml exist). Maybe you could modify this script to check that it looks like the new format i.e. tests_ethernet_nm.yaml and tests_ethernet_initscripts.yaml

Reply:
The scripts supports an ignore list: https://github.com/linux-system-roles/network/blob/master/tests/ensure_non_running_provider.py#L16 Once everything is ported, the script can be removed.

Reply:
> @a-mere-peasant Looks like your build is failing because you renamed this file and the molecule tests in `molecule/default/molecule.yml` are hardcoded to look for it
> 
> https://github.com/linux-system-roles/network/blob/51fa1146b4eae53606322dbdca32e1386cce5242/molecule/default/molecule.yml#L26

Good catch, I guess for the default tests it makes sense to keep the old test and and add two more playbooks for nm and initscripts that set the provider and include the original playbook.